we get thousands of these "warnings" with one actual error in the middle; 
save the human time by not having to hunt for the one error

--- dnf-3.0.3/dnf/base.py~	2018-07-12 09:49:08.000000000 +0000
+++ dnf-3.0.3/dnf/base.py	2018-12-21 16:40:27.745216326 +0000
@@ -2327,4 +2327,3 @@
 def _msg_installed(pkg):
     name = ucd(pkg)
     msg = _('Package %s is already installed, skipping.')
-    logger.warning(msg, name)
